src/cfnlint/rules/resources/lmbd/FunctionMemorySize.py

```diff
@@ -31,17 +31,17 @@ def check_value(self, value, path):
 
         message = 'You must specify a value that is greater than or equal to 128, ' \
                   'and it must be a multiple of 64. You cannot specify a size ' \
-                  'larger than 1536. The default value is 128 MB at {0}'
+                  'larger than 3008. Error at {0}'
 
         try:
             value = int(value)
 
-            if value < 128 or value > 1536:
-                matches.append(RuleMatch(path, message.format(value, ('/'.join(path)))))
+            if value < 128 or value > 3008:
```

Comment:
Isn't it an idea to make them constants? Or even configurable? Instead of changing it in 4 places

Reply:
fixed in most recent pull request